### DISCUSSION TITLE
Notifications

### DISCUSSION THREAD

**Discussion by rubiin** [prasanthrangan/hyprdots#1805] [General]
Is there a way to see missed notifications?

**Comment by kRHYME7** [comment]
Maybe? As long as it tried to pop up.

**Reply by rubiin** [reply]
I was guessing something like swaync that has list of all past notofication.

**Reply by rubiin** [reply]
Also can you help me with this style in rofi 

![](https://raw.githubusercontent.com/adi1090x/rofi/master/previews/launchers/type-3/2.png)

**Reply by kRHYME7** [reply]
sure this one has 
` -fullscreen   ` flag or in *.rasi ` fullscreen : true; 


![image](https://github.com/user-attachments/assets/73c9a990-0236-4f59-a90d-6c31c1b8f1f2)


you can also just make the width and height into 100% so it will respect the other layers like wayabr  
![image](https://github.com/user-attachments/assets/8433b665-b2c4-4ee4-a8ac-2e14f7f714b0)
![image](https://github.com/user-attachments/assets/9b6c4683-9c1e-43de-937f-341360778cbe)




Check this out https://man.archlinux.org/man/extra/rofi-wayland/rofi.1.en

**Reply by rubiin** [reply]
Got it. This style plus way corner combo makes me feel like KDE corners
![image](https://github.com/user-attachments/assets/12107db1-f389-464c-84b7-8eae95994810)

**Reply by rubiin** [reply]
https://github.com/adi1090x/rofi
